BookingProject/BookingWebAPI/BookingWebApi.Application/Mapper/MapperProfile.cs

```diff
@@ -15,6 +16,12 @@ public MapperProfile()
         {
             CreateMap<Apartament, ApartamentPostDto>().ReverseMap();
             CreateMap<Apartament, ApartamentGetDto>().ReverseMap();
+            CreateMap<AppUser, AppUserMigrationDto>()
+                .ForMember(x => x.Id, y => y.MapFrom(src => src.ExternalId))
```

Comment:
Impossible. Imagine situation: User Id in your system is string Guid. User Id in their system is int, for example 123. Do you really want to store them in the same column? I doubt.
Better create a separate field in AppUser entity

Reply:
You can map Id here as `src => Guid.NewGuid().ToString()` to generate new id for your inserted user

Reply:
Fixed